R/plot-heatmap.R

```diff
@@ -42,6 +47,8 @@ plot_heatmap <- function(mm,
     label_level = 1,
     max_label_len = 60,
     sample_label = FALSE,
+    scale_by_row = FALSE,
+    use_gghue = TRUE,
```

Comment:
Thanks. I think we can add a new argument `annotation_col` like the argument `col` in `HeatmapAnnotation()`  to specify the annotation colors, so that the colors are not limited to hue platte. What do you think?

Reply:
Sure. It's better to leave that open for more choices. I will use `annotation_col` to get a named vector for color selection.

Reply:
You can add the motivation of this pull requst in `NEWS.md` so that it's easy for users to see what's changed. Add your item at the top of the file, and the news item should end with (@yourGithubUsername, #the_issue_number). Thank you very much.